src/bin/tui/table.rs

```diff
@@ -492,7 +492,7 @@ impl<T: TableViewItem<H>, H: Eq + Hash + Copy + Clone + 'static> TableView<T, H>
 	/// Returns the index of the currently selected item within the underlying
 	/// storage vector.
 	pub fn item(&self) -> Option<usize> {
-		if self.items.is_empty() {
+		if self.items.is_empty() || self.focus > self.rows_to_items.len() {
```

Comment:
Should this actually be `>=`?

I just happened to see the following panic locally - 

```
thread 'main' panicked at 'index out of bounds: the len is 15 but the index is 15': /rustc/4560ea788cb760f0a34127156c78e2552949f734/src/libcore/slice/mod.rs:2717stack backtrace:
```

Reply:
Yes you're right I think it should. I can do a fix for that now.

Reply:
If its just this one line change I think we can get it in for `beta.2`.
:+1: